modules/registry.js

```diff
@@ -0,0 +1,83 @@
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define(['underscore', 'backbone'], factory);
+  } else if (typeof exports === 'object') {
+    module.exports = factory(require('underscore'), require('backbone'));
+  } else {
+    root.Torso = root.Torso || {};
+    root.Torso.registry = factory(root._, root.Backbone);
+  }
+}(this, function(_, Backbone) {
+  'use strict';
+
+  /**
+   * Registry of intantiated Torso objects
+   * @module    Torso
+   * @class     registry
+   * @constructor
+   * @author kent.willis@vecna.com
+   */
+  var registry = new (Backbone.Model.extend({
+
+    initialize: function() {
+      this.cells = {};
+      this.models = {};
+      this.services = {};
+      this.views = {};
+    },
+
+    modelInitialized: function(model) {
+      this.__initialize(model, this.models);
+    },
+
+    cellInitialized: function(cell) {
+      this.__initialize(cell, this.cells);
+    },
+
+    viewInitialized: function(view) {
+      this.__initialize(view, this.views);
+    },
+
+    serviceInitialized: function(service) {
+      this.__initialize(service, this.services);
+    },
+
+    __initialize: function(obj, cache) {
+      cache[obj.cid] = obj;
+      this.listenTo(obj, 'before-dispose', function() {
+        delete cache[obj.cid];
+      });
+    },
+
+    disposeAll: function() {
```

Comment:
I like all of the disposeAll methods.

Docs would be nice for the methods in the registry, though.